src/app/pages/registration/register-form/register-form.component.html

```diff
@@ -0,0 +1,150 @@
+<div class="container">
+  <div class="details">
+    <div class="information">
+      <div class="left-section">
+        <h4>Create a free account today!</h4>
+        <form [formGroup]="registerForm" (ngSubmit)="onSubmit(registerForm.value)">
```

Comment:
Hey @akram256, I notice the positioning of the error messages is more shifted to the left, I believe the design can look better if the messages were on the same level with the form Control inputs.

Reply:
Hey @CryceTruly. Thank you so much. I worked on this